packages/github/src/components/icons/PR.tsx

```diff
@@ -17,4 +19,70 @@ export const PR = forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(({ classNam
   </svg>
 ))
 
-PR.displayName = 'PR'
+PROpen.displayName = 'PROpen'
+
+export const PRClosed = forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(({ className }, ref) => (
+  <svg
+    className={cn('inline-block', className)}
+    fill="currentColor"
+    height="1em"
+    ref={ref}
+    stroke="currentColor"
+    strokeWidth={0.5}
+    viewBox="0 0 16 16"
+    width="1em"
+  >
+    <path d="M3.25 1A2.25 2.25 0 0 1 4 5.372v5.256a2.251 2.251 0 1 1-1.5 0V5.372A2.251 2.251 0 0 1 3.25 1Zm9.5 5.5a.75.75 0 0 1 .75.75v3.378a2.251 2.251 0 1 1-1.5 0V7.25a.75.75 0 0 1 .75-.75Zm-2.03-5.273a.75.75 0 0 1 1.06 0l.97.97.97-.97a.748.748 0 0 1 1.265.332.75.75 0 0 1-.205.729l-.97.97.97.97a.751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018l-.97-.97-.97.97a.749.749 0 0 1-1.275-.326.749.749 0 0 1 .215-.734l.97-.97-.97-.97a.75.75 0 0 1 0-1.06ZM2.5 3.25a.75.75 0 1 0 1.5 0 .75.75 0 0 0-1.5 0ZM3.25 12a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5Zm9.5 0a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5Z" />
+  </svg>
+))
+
+PRClosed.displayName = 'PRClosed'
+
+export const PRMerged = forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(({ className }, ref) => (
+  <svg
+    className={cn('inline-block', className)}
+    fill="currentColor"
+    height="1em"
+    ref={ref}
+    stroke="currentColor"
+    strokeWidth={0.5}
+    viewBox="0 0 16 16"
+    width="1em"
+  >
+    <path d="M5.45 5.154A4.25 4.25 0 0 0 9.25 7.5h1.378a2.251 2.251 0 1 1 0 1.5H9.25A5.734 5.734 0 0 1 5 7.123v3.505a2.25 2.25 0 1 1-1.5 0V5.372a2.25 2.25 0 1 1 1.95-.218ZM4.25 13.5a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5Zm8.5-4.5a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM5 3.25a.75.75 0 1 0 0 .005V3.25Z" />
+  </svg>
+))
+
+PRMerged.displayName = 'PRMerged'
+
+export const PRDraft = forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(({ className }, ref) => (
+  <svg
+    className={cn('inline-block', className)}
+    fill="currentColor"
+    height="1em"
+    ref={ref}
+    stroke="currentColor"
+    strokeWidth={0.5}
+    viewBox="0 0 16 16"
+    width="1em"
+  >
+    <path d="M3.25 1A2.25 2.25 0 0 1 4 5.372v5.256a2.251 2.251 0 1 1-1.5 0V5.372A2.251 2.251 0 0 1 3.25 1Zm9.5 14a2.25 2.25 0 1 1 0-4.5 2.25 2.25 0 0 1 0 4.5ZM2.5 3.25a.75.75 0 1 0 1.5 0 .75.75 0 0 0-1.5 0ZM3.25 12a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5Zm9.5 0a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5ZM14 7.5a1.25 1.25 0 1 1-2.5 0 1.25 1.25 0 0 1 2.5 0Zm0-4.25a1.25 1.25 0 1 1-2.5 0 1.25 1.25 0 0 1 2.5 0Z" />
+  </svg>
+))
+
+PRDraft.displayName = 'PRDraft'
+
+export const PRIcon = ({ className, status }: { className: string; status: string }) => {
```

Comment:
nit: we can improve type safety and add default handling with fixed values 

.. oh, to keep it open but have a nice auto complete we can do it like so:

```suggestion
type PRStatus = ('closed' | 'draft' | 'merged' | 'open' | string) & {}

export const PRIcon = ({ className, status }: { className: string; status: PRStatus }) => {
```